Examples/iOS/LoopbackRecording/LoopbackRecording/ViewController.swift

```diff
@@ -71,32 +71,33 @@ class ViewController: UIViewController {
     func setUpAudio() {
 
         do {
-            let audioSession = AVAudioSession.sharedInstance()
-            try audioSession.setActive(true)
-            try audioSession.setCategory(.playAndRecord, mode: .default, options: [.defaultToSpeaker, .mixWithOthers])
+            AKSettings.audioInputEnabled = true
+            AKSettings.defaultToSpeaker = true
+            AKSettings.sampleRate = AudioKit.engine.inputNode.inputFormat(forBus: 0).sampleRate
+            try AKSettings.setSession(category: .playAndRecord)
+
 
             // Measurement mode can have an effect on latency.  But you end up having to boost everything.
             // It's a must if you want accurate recordings.  It turns of the os input processing.
             // Uncomment / Experiment
-            /*
 
-            try audioSession.setMode(AVAudioSessionModeMeasurement)
+            // try AKSettings.session.setMode(AVAudioSession.Mode.measurement)
 
-             */
 
         } catch {
             fatalError(error.localizedDescription)
         }
 
-        // Make connections, using engine.outputNode so that I can skip AudioKit.start()
-        [metronome, player] >>> mixer >>> AudioKit.engine.outputNode
+        // Make connections
+        metronome.connect(to: mixer.inputNode)
+        player.connect(to: mixer.inputNode)
```

Comment:
I think these can just be `node.connect(to: otherNode)`. The connect function will end up using inputNode implicitly.